features/core.feature

```diff
@@ -20,6 +20,20 @@ Feature: Basic reading and writing to a journal
         When we run "jrnl -n 1"
         Then the output should contain "2013-07-23 09:00 A cold and stormy day."
 
+    Scenario: Writing an empty entry from the editor
+        Given we use the config "editor.yaml"
+        When we open the editor and enter ""
+        Then we should see the message "[Nothing saved to file]"
+
+    Scenario: Writing an empty entry from the command line
```

Comment:
If we decide we don't care about this last test, we can just delete this block and merge it. I don't think this test is super important and likely won't spend more time debugging it.